packages/typespec-vscode/src/extension.ts

```diff
@@ -276,3 +284,49 @@ function showStartUpMessages(stateManager: ExtensionStateManager) {
     stateManager.cleanUpStartUpMessage(workspaceFolder.uri.fsPath);
   });
 }
+
+async function sendTelemetryDataForEmitInTspConfig(activityId: string) {
```

Comment:
I thikn this is now out of scope of what we are approved to collect. You are fetching user files and uploading user content.

Reply:
Agreed. Let's close this PR and revisit this later.

Reply:
I think we discussed it in the channel and confirmed it's fine to collect them. Anything change again? thanks. @timotheeguerin , @allenjzhang, @mario-guerra 
https://teams.microsoft.com/l/message/19:meeting_MDBiY2NmYWYtOTk2Yi00MDE5LTgxOWUtZDg0NGU2NjBiMjM2@thread.v2/1742480539667?context=%7B%22contextType%22%3A%22chat%22%7D

Reply:
we agreed on the emitter for the exiting flow and for things that flow through the extension/lsp(as long as its done on the vscode side) here you are deliberately spying on user files